sample/HttpTrigger-Powershell/run.ps1

```diff
@@ -1,10 +1,10 @@
-if ($Env:req_query_name) 
```

Comment:
Does this other syntax also still work (explicit reference of $Env?) I.e., are there breaking changes?